### ISSUE TITLE
Any fix for KeyError: 'depots'?

### ISSUE THREAD

**Issue by ghost** [ericwoolard/CS-GO-Update-Notifier#1]
Has there been any fix or updates for this issue? Haven't been able to start bot up for while because of this issue

**Comment by ghost** [comment]
Found this, may help? 
https://github.com/ValvePython/steamctl/issues/7

**Comment by aquaismissing** [comment]
> Found this, may help?
> [ValvePython/steamctl#7](https://github.com/ValvePython/steamctl/issues/7)

Nah, the thing is that you can't get depot info with `client.anonymous_login()`, just change it to:
 `client.login(username='', password='', auth_code='')` 
or 
`client.login(username='', password='', two_factor_code='')`

I would recommend you to open an empty steam account with 2FA and Email auth turned off, so you can just use:
`client.login(username='', password='')`

Anyway, more info about `login()` here:
https://github.com/ValvePython/steam/blob/44d0ef5cc3f420205e3d893abb362a507dd93af2/steam/client/__init__.py#L484

**Comment by ghost** [comment]
> > Found this, may help?
> > [ValvePython/steamctl#7](https://github.com/ValvePython/steamctl/issues/7)
> 
> Nah, the thing is that you can't get depot info with `client.anonymous_login()`, just change it to:
> `client.login(username='', password='', auth_code='')`
> or
> `client.login(username='', password='', two_factor_code='')`
> 
> I would recommend you to open an empty steam account with 2FA and Email auth turned off, so you can just use:
> `client.login(username='', password='')`
> 
> Anyway, more info about `login()` here:
> https://github.com/ValvePython/steam/blob/44d0ef5cc3f420205e3d893abb362a507dd93af2/steam/client/__init__.py#L484

Issue with this is a still think it needs an update, still errors out and times out etc

**Comment by ericwoolard** [comment]
Updated with suggested login.

**Comment by ghost** [comment]
> Updated with suggested login.

(this is with providing a steam account in the .py as well)
For me after running with the fix above and your new one it just says "none" after running then a few seconds later crashes
![image](https://user-images.githubusercontent.com/16860411/107601268-d7fb7680-6beb-11eb-836d-090eb4842b95.png)
